ferretdb/ferretdb.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ferretdb
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/FerretDB/FerretDB/internal/clientconn"
+	"github.com/FerretDB/FerretDB/internal/handlers/common/register"
+	"github.com/FerretDB/FerretDB/internal/util/debug"
+	"github.com/FerretDB/FerretDB/internal/util/logging"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/version"
+)
+
+// config is a FerretDB library config.
+var config Config
+
+// Config ConnectionString contains a backend connection string.
+type Config struct {
+	ConnectionString string
+}
+
+// GetConnectionString returns the backend connection string.
+func GetConnectionString() string {
+	return config.ConnectionString
+}
+
+// Run runs the FerretDB proxy as a library.
+func Run(ctx context.Context, conf Config) error {
```

Comment:
```suggestion
func Run(ctx context.Context, conf Config) (string, error) {
```

It could return a MongoDB connection string